heapster-saw/examples/clearbufs.saw

```diff
@@ -4,12 +4,14 @@ env <- heapster_init_env_from_file "clearbufs.sawcore" "clearbufs.bc";
 // Integer types
 heapster_define_perm env "int64" " " "llvmptr 64" "exists x:bv 64.eq(llvmword(x))";
 
+// FIXME: get reachability perms working again!
```

Comment:
This sort of thing would also be worth mentioning in an issue about re-enabling Heapster tests (should you choose to file one), along with the other FIXMEs in the `heapster-saw/examples` subdirectory.